tests/handlers.rs

```diff
@@ -146,6 +146,72 @@ speculate! {
             }
         }
 
+        describe "when incomplete option" {
+            before {
+                let uri = flux_fixture_uri("incomplete_option");
+            }
+
+            after {
+                close_file(uri, &mut handler);
+            }
+
+            it "returns an error" {
```

Comment:
is this still the case after the flux version was rolled?

Reply:
it does return an error because the incomplete option is not valid but it no longer panics and instead handles it gracefully